### PR TITLE
Added idd.py and created a bazel wrapper for it

### PR DESCRIPTION
**Pull request by ArthurRab**

### CODE REVIEW

File: contrib/idd.bzl [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    name = "name",
+    image1 = "@<image1>//image",
+    image2 = ""hopefully_identical_image.tar",
+    args = ["-v", "-d"]
+)
+"""
+idd = rule(
+    implementation = _impl,
+    attrs = {
+        "image1": attr.label(mandatory = True, allow_files = True),
```
Comment (xingao267) [inline contrib/idd.bzl]: use single_file

Comment (xingao267) [reply contrib/idd.bzl]: Also use allow_files = container_filetype, like https://github.com/bazelbuild/rules_docker/blob/master/container/image.bzl#L351 if possible.

Comment (ArthurRab) [reply contrib/idd.bzl]: Added both

**Comment by xingao267** [comment]
Add test cases that check the output script.

File: contrib/idd.bzl [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    image2: Image target or image tarball file (from docker save) - second image to compare
+    args: (optional) list of strings - arguments to apply to idd.py call 
+                                        refer to idd.py docs for more info
+
+Ex.
+
+idd(
+    name = "name",
+    image1 = "@<image1>//image",
+    image2 = ""hopefully_identical_image.tar",
```
Comment (xingao267) [inline contrib/idd.bzl]: aha, one extra quotation mark

Comment (ArthurRab) [reply contrib/idd.bzl]: Oops

**Pull request merged**